File: 0169-majority-element/0169-majority-element.c
int majorityElement(int* nums, int numsSize) {
    int co=0;
    int can=nums[0];
    for(int i=0;i<numsSize;i++)
    {
        if(co==0)
        {
            can=nums[i];
        }
        if(can==nums[i])
        {
            co++;
        }
        else
        {
            co--;
        }
    }
    return can;
}

File: 0169-majority-element/0169-majority-element.java
class Solution {
    public int majorityElement(int[] nums) {
        int cm=nums[0];
        int m=0;
        for(int i=0;i<nums.length;i++)
        {
            if(m==0)
            {
                cm=nums[i];
            }
            if(nums[i]==cm)
            {
                m++;
            }
            else
            {
                m--;
            }
            
        }
        return cm;
    }
}